content/actions/reference/context-and-expression-syntax-for-github-actions.md

```diff
@@ -93,7 +93,7 @@ The `github` context contains information about the workflow run and the event t
 | `github.event_path` | `string` | The path to the full event webhook payload on the runner. |
 | `github.head_ref` | `string` | The `head_ref` or source branch of the pull request in a workflow run. This property is only available when the event that triggers a workflow run is a `pull_request`. |
 | `github.job` | `string` | The [`job_id`](/actions/reference/workflow-syntax-for-github-actions#jobsjob_id) of the current job. |
-| `github.ref` | `string` | The branch or tag ref that triggered the workflow run. |
+| `github.ref` | `string` | The branch or tag ref that triggered the workflow run in the form of `refs/heads/<BRANCH_NAME>`. |
```

Comment:
Thanks for this PR, @tillepille! Your change covers the format for branches, but I'll also add the format for tags too 🙂 

```suggestion
| `github.ref` | `string` | The branch or tag ref that triggered the workflow run. For branches this in the format  `refs/heads/<branch_name>`, and for tags it is `refs/tags/<tag_name>`. |
```

Reply:
I didn't test that. Thank you @lucascosti  !

Reply:
No worries, thanks a lot for the contribution! 👍